src/components/data/listing/Listing.js

```diff
@@ -213,12 +211,20 @@ function Listing(props) {
                 page,
                 uploadsCompleted,
             ],
-            { force: true }
+            { exact: true }
         );
 
-    const [removeResources, { status: removeResourceStatus }] = useMutation(
-        deleteResources,
-        {
+    const { defaultsMappingError, isFetchingDefaultsMapping } = useQuery({
```

Comment:
This doesn't look right 🤔 
But I noticed that's how it looked before this PR, too.

Maybe we should add a `replyOnce` test error for this endpoint in this listing's Story, to test if this is working as expected.

Reply:
Good catch. The return object is not spread correctly. I also think we have don't block the listing just because we can't fetch IL mappings. So changed the error handling logic to just show an error announcer.